docs/api.md

```diff
@@ -453,3 +453,11 @@ put any value on this annotation **other than `metal3.io/capm3`**. Please make
 sure that you remove the annotation  **only if the value of the annotation is
 not `metal3.io/capm3`, but another value that you have provided**. Removing the
 annotation will enable the reconciliation again.
+
+## Detaching hosts
+
+It is possible to pause the management of a BareMetalHost object by adding
+an annotation `baremetalhost.metal3.io/detached`. This removes the host from
+the provisioner, and prevents any further state changes, but it does not
+fully pause reconciliation (so the status field can be updated, and the
+OperationalStatus field will reflect the `detached` status).
```

Comment:
"possible to pause" and "does not fully pause" are a little confusing.
Looking at the code, maybe we can reword to say that we can "watch the state of the BMH without being able to directly manage it."

Reply:
"possible to pause" is a copy/paste typo, and I like your suggestion @sadasu thanks!